stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -678,8 +678,10 @@ public func == <Element : Hashable>(lhs: Set<Element>, rhs: Set<Element>) -> Boo
   case (.Native(let lhsNativeOwner), .Native(let rhsNativeOwner)):
     let lhsNative = lhsNativeOwner.nativeStorage
     let rhsNative = rhsNativeOwner.nativeStorage
-    // FIXME(performance): early exit if lhs and rhs reference the same
-    // storage?
+
+    if unsafeAddressOf(lhsNativeOwner) == unsafeAddressOf(rhsNativeOwner) {
```

Comment:
I don't think that's right, this code would compare addresses of the `let` bindings.

Does `===` work?